src/crimson/os/seastore/transaction.h

```diff
@@ -244,6 +246,8 @@ class Transaction {
   OrderingHandle handle;
 
   const src_t src;
+
+  std::chrono::steady_clock::time_point tp;
```

Comment:
could use a better name than `tp`. how about `begin_time` or `begin_timestamp` ?

Reply:
And move it to `internal_context_t` will be more appropriate.

Reply:
repeat_with_onode not use internal_context_t.

Reply:
A local variable in that function can work.

Reply:
done